sample/library/src/test/kotlin/app/cash/paraphrase/sample/library/FormattedResourcesTest.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2023 Cash App
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.paraphrase.sample.library
+
+import androidx.annotation.StringRes
+import com.google.common.truth.Truth.assertThat
+import com.ibm.icu.text.MessageFormat
+import java.time.LocalDate
+import java.time.LocalTime
+import java.time.Month
+import java.util.Locale
+import org.junit.Test
+
+class FormattedResourcesTest {
```

Comment:
Is the idea here just to show people how they might override the `FormattedResources` type in tests?

Reply:
This tests that date/time resolution works in a JVM test. This test fails on main because of the ICU4J/`android.icu` discrepancy.